draft-packages/tooltip/docs/Tooltip.stories.tsx

```diff
@@ -1,20 +1,28 @@
 import * as React from "react"
 
 import { Paragraph } from "@kaizen/component-library"
+import { MenuItem, MenuContent } from "@kaizen/draft-menu"
 import { Tag } from "@kaizen/draft-tag"
+import { SplitButton } from "@kaizen/draft-split-button"
 import { withDesign } from "storybook-addon-designs"
 import { Button } from "@kaizen/draft-button"
 import { Tooltip } from "@kaizen/draft-tooltip"
+import isChromatic from "chromatic/isChromatic"
 import { figmaEmbed } from "../../../storybook/helpers"
 
+const excludeFromChromatic = Story => {
+  if (!isChromatic()) return <Story />
+  return null
+}
```

Comment:
Why exclude the tooltip stories from Chromatic? If anything, I would expect it would be good to have screenshot coverage of this stacking issue, so we avoid a regression.